Dockerfile.spot

```diff
@@ -24,13 +34,7 @@ COPY . /azure-cli
 # 1. Build packages and store in tmp dir
 # 2. Install the cli and the other command modules that weren't included
 # 3. Temporary fix - install azure-nspkg to remove import of pkg_resources in azure/__init__.py (to improve performance)
-RUN apt-get install -y ca-certificates curl openssl git \
- && apt-get install -y gcc make libffi-dev \
```

Comment:
Are there any corresponding installations to this or do we not need these installations in the spot file?

Reply:
I have not changed which packages are installed with this change. I've only reorganized which layer they are installed in to improve Docker cache performance.

Reply:
So does the docker spot instances still have these installations that were removed?

Like `apt-get install -y gcc make libffi-dev `

Reply:
They were not removed, they were just moved to an earlier layer:

https://github.com/Azure/azure-cli/pull/8041/files#diff-eb2083d21ce2d35b5c085eea3cff77baR15